vendor-frontend/src/views/Dashboard.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="container">
+    <p>Dashboard</p>
+    <p>{{info.id}}</p>
+    <p>{{info.name}}</p>
+    <p>{{info.city}}</p>
+    <p>{{info.address}}</p>
+      
+    <ul id="example-1">
+        <li v-for="(tray, index) in info.trays" :key="index">
+            {{tray.description}} - {{index}}
+        </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+    import axios from 'axios'
+
+    export default {
+        data () {
+            return {       
+                info: "",
+            }   
+        },
+        mounted(){
+            axios
+            .get("http://127.0.0.1:8000/vendors/1")
```

Comment:
does this need to be hardcoded? Or can it be referenced somewhere?

Reply:
Oh, I can change that part, I only created this pull request to add the CORS-middlewares for George.

Reply:
oh okay sounds good @xinruili07  if you can change it, lets do it

Reply:
bump